### ISSUE TITLE
The "Read More..." help text font is ugly

### ISSUE THREAD

**Issue by kumarharsh** [microsoft/vscode#76280]
Version: 1.36.0-insider (user setup)
Commit: b7265738444f1eee05762341b4361ee6452a4154
Date: 2019-06-27T05:22:06.378Z
Electron: 4.2.5
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

Steps to Reproduce:

1. Go to the Settings page, and into the JSON editing mode
2. Type something to bring up the autocomplete suggestions, and then press <kbd>Ctrl</kbd>+<kbd>Space</kbd>

![image](https://user-images.githubusercontent.com/760112/60322625-8d3cd480-999d-11e9-823c-3bccffbe23c2.png)

This documentation should have the same font as the usual workbench text (Segoe or whatever), but it's too bold

Does this issue occur when all extensions are disabled?: Yes

**Comment by roblourens** [comment]
Can you include some of the workbench text in your screenshot to compare?

**Comment by kumarharsh** [comment]
Context menu and other workbench UI:
![image](https://user-images.githubusercontent.com/760112/60419889-fc1b6700-9c03-11e9-936c-6cff8dd9f639.png)

The hints on top of code:
![image](https://user-images.githubusercontent.com/760112/60419909-09385600-9c04-11e9-9e59-8b89616cb215.png)

Autocomplete from other places:
![image](https://user-images.githubusercontent.com/760112/60419992-3b49b800-9c04-11e9-91f1-0540fcd78312.png)


Seems like all autocomplete help text are bolded. I can't even inspect these nodes as they close before I can use DevTools inspector on them, and I can't find out which DOM node they attach under (they're not under `body`), so I can't use DOM breakpoints either...

**Comment by roblourens** [comment]
It does look bold, I don't know where this comes from...

**Comment by jrieken** [comment]
Suggestions use the editor font and that doesn't look like the default. What's the settings  value of `editor.fontFamily`?

**Comment by kumarharsh** [comment]
Oh, I get what's going wrong.

I've set the Iosevka SS07 font as the editor font. The full list is:
```
"Iosevka SS07", "Fira Code", "Source Code Pro", 'Courier New', monospace
```

But since the font is slightly thinner than my liking, I've set the Editor font family to be bold. Now, it didn't dawn on me that this was causing the issue because the *Editor font weight* applies to more places than *Editor font family* - case in point, the screenshots above. As can be seen in the screenshots, the font family of the description is *definitely not* Iosevka, but *probably* Segoe(?)

I can close this issue, but maybe it might be worthwhile to clarify in the settings screen that the font weight applies to more places in the UI than the font family.
